<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">

    <style>
        /* Existing CSS styles */
        body {
            font-family: Calibri, sans-serif;
            font-size: 12pt;
            margin: 20px 10%; /* Reduced top margin */
            padding-top: 120px; /* Increase padding to match the larger header box */
        }

        /* Fixed header styles */
        .fixed-header {
            position: fixed;
            top: 0;
            left: 0;
            width: 100%;
            background-color: #7AB2D3; /* Match the original background color */
            padding: 0;
            z-index: 1000;
            text-align: center;
            box-shadow: 0px 4px 2px -2px gray; /* Add a shadow for clearer separation */
        }

        .fixed-header h1 {
            margin: 0;
            padding: 30px; /* Increase the padding to make the box bigger */
            font-size: 50px;
            font-weight: bold;
            color: white; /* Change the text color to white */
            text-align: center;
            line-height: 1.5; /* Adjust the line-height for better spacing */
        }

        /* Responsive styles for smaller screens */
        @media (max-width: 1024px) {
            body {
                margin: 10px 7%; /* Reduce top margin for smaller screens */
            }
        }

        @media (max-width: 768px) {
            body {
                margin: 5px 5%; /* Further reduce top margin on smaller devices */
            }
        }

        @media (max-width: 480px) {
            body {
                margin: 5px 2%; /* Minimal margins for mobile devices */
            }

            .fixed-header h1 {
                font-size: 24px; /* Adjust header font size for small screens */
                padding: 20px; /* Adjust padding for smaller screens */
            }
        }
		footer {
    background-color: #f1f1f1; /* Light grey background */
    text-align: center; /* Centered text */
    padding: 10px; /* Padding around the text */
    position: relative; /* Positioned relative to the parent */
    bottom: 0; /* Stick to the bottom */
    width: 100%; /* Full width */
}

    </style>
	
    <title></title>
</head>

<body lang="EN-US" spellcheck="false" style="overflow-wrap: break-word;">

    <!-- Fixed header section -->
    <div class="fixed-header">
        <h1>Module 3</h1>
    </div>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:12.0pt;margin-bottom:8.0pt;font-size:11.0pt;text-align:center;'><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";'>Nanopore based Data Analysis and AMR detection</span></strong></p>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:15.0pt;font-family:"Calibri",sans-serif;'>
<ol start="1" style="margin-bottom:0cm;list-style-type: decimal;margin-left:0px;padding-left: 0;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:15.0pt;font-family:"Calibri",sans-serif;'><strong><em><span style='font-family:"Times New Roman",serif;font-size:20px;color:black;'>Prerequisite and Software Setup<span></em></strong></li>
    </ol>
</div>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;line-height:normal;'><strong><span style='font-size:16px;font-family:"Times New Roman";'>1.1 Ubuntu: Installation (Windows)</span></strong></p>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='font-family:"Times New Roman",serif;font-size:16px;'>Open Windows Powershell and run as administrator mode. Tip: search &ldquo;powershell&rdquo; in Windows search</span></li>
    </ul>
</div>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='font-family:"Times New Roman",serif;font-size:16px;'>To install WSL, run the command:</span></li>
    </ul>
</div>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;height: 14.95pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><code style='font-family:"Courier New";'><span style="font-size:13px;font-family:Calibri;">wsl --install</span></code></p>
            </td>
        </tr>
    </tbody>
</table>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='font-family:"Times New Roman",serif;font-size:16px;'>To set your root password, enter a new user and password</span></li>
    </ul>
</div>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='font-family:"Times New Roman",serif;font-size:16px;'>When installation is finished, reboot the machine.</span></li>
    </ul>
</div>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;'><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>1.2 Installing&nbsp;</span></strong><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>Custom Conda Environment (</span></strong><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>Miniconda)</span></strong></p>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='font-family:"Times New Roman",serif;font-size:16px;'>Open Ubuntu terminal from Windows Start menu. Click on the &ldquo;Windows&rdquo; button and search for &ldquo;ubuntu&rdquo;. Clicking on the application will open an Ubuntu terminal, which is a LINUX terminal</span></li>
    </ul>
</div>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='font-family:"Times New Roman",serif;font-size:16px;'>If prompted, enter your username and password from step 1.</span></li>
    </ul>
</div>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='font-family:"Times New Roman",serif;font-size:16px;'>In this workshop, we will be using Miniconda&nbsp;</span><span style='font-family:"Times New Roman",serif;font-size:16px;color:#222222;'>(recommended as it is relatively small and quick to install)</span><span style='font-family:"Times New Roman",serif;font-size:16px;'>&nbsp;as a package manager to facilitate replication of working environments with all necessary software packages. To learn more about Conda, see:&nbsp;</span><a href="https://docs.conda.io/projects/conda/en/latest/user-guide/index.html"><span style='font-family:"Times New Roman",serif;font-size:16px;'>https://docs.conda.io/projects/conda/en/latest/user-guide/index.html</span></a><span style="color:blue;text-decoration:underline;"><span style='font-family:"Times New Roman",serif;font-size:16px;'>&nbsp;</span></span><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>&nbsp;</span></li>
    </ul>
</div>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='font-family:"Times New Roman",serif;font-size:16px;'>Download the latest Miniconda installer for Linux:</span></li>
    </ul>
</div>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh</span></p>
            </td>
        </tr>
    </tbody>
</table>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>Run the installer:</span></li>
    </ul>
</div>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;height: 17.5pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>bash Miniconda3-latest-Linux-x86_64.sh</span></p>
            </td>
        </tr>
    </tbody>
</table>
<ul style="list-style-type: square;margin-left: 26px;">
    <li>Follow the prompts to complete the installation</li>
    <li>Close and reopen the terminal to apply changes</li>
    <li>If you have followed the instructions properly, you should see (base) at the beginning of the command line. This means that Miniconda is already activated.</li>
</ul>
<p style='margin-right:0cm;margin-left:36.0pt;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;'><img width="194" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAMIAAAAcCAIAAAC8t/c5AAAACXBIWXMAAAsTAAALEwEAmpwYAAAAtGVYSWZJSSoACAAAAAYAEgEDAAEAAAABAAAAGgEFAAEAAABWAAAAGwEFAAEAAABeAAAAKAEDAAEAAAACAAAAEwIDAAEAAAABAAAAaYcEAAEAAABmAAAAAAAAAEgAAAABAAAASAAAAAEAAAAGAACQBwAEAAAAMDIxMAGRBwAEAAAAAQIDAACgBwAEAAAAMDEwMAGgAwABAAAA//8AAAKgBAABAAAAwgAAAAOgBAABAAAAHAAAAAAAAADNp1TQAAAF1ElEQVR4nO1avU7rShBep05eIKliRYgy6Ull0RCJGhehQykQzXFPFFrk0yCKtKQwHRJSaJAr0sclQkemCg9wXmCvZnZt79rrjQ25EA7+KsLsz+zMNz+7iUEpJRUqfAy1D86vUIEQYtTr9a/WocK3R5WNKmwAFY0qbAAVjSpsmkb27G0x7sUfe+PF29vMNgzy6YCtF+PeV2z9ERiGPXuLMLPJD6SRPXtzrfDxPiBbgOD+MTRH85sjsn0sESNNktqz1colTqvZbLZag2nb/TlM4jQy7JlrEd/pT5Zb8YxEl5O+4xPLndnfIyEZhn3jWr7TGnpgQEqXF33Ht04ho4opikFOtAam/RTEg6cGiCJcOqkYfCSur1w23j5We6ZbOVclBY2YCYjvMBNsCag3RCLdfElVLYsjJJFoQEq9B9/cP+yyj+F00ERAoiKj+eop9iWDj2ksRrwUJLn5iETTm83mw0F6Lh9p2Dfzkek7zb3JklK6nOzFO4aoQYttAeI4fbZjxRzfcldyrvUdPqU5mIYZqUyj7vkpkOj4NsdE3YTVcpbWREmG6UkERFJxclrKcPt7GhLr9Jx7oghwVVhNWF1aXBN/GilbbbVyLULM0VzUm+CsAyuc/k4b8PlPaHZ2U//ERDWYhubo6nxt82cYPXBOOD27SJoNb8h5oKCyzo+qlX1nb7LkinlDVOuXMm6Xkz3HJ3nSmmH0DvdN4j946m9FLHc171xjvUdCxnwE7l+RM4mtSZSAdAUBGoXgdUewGkrno1cuZeGZzZl0Obn2ibl/WLLXNkdzUDoKxMRhRm/8JMQfi7B4X42UBXar5fhCXmFxTcCBB6yrTLXYwcsrae9kg4DS5f1jCAcrGCBZMmbQGy9cK5wOjnP8mEGXuf023ZIS60DXkarOA9kIlwv/POfNS+q97FRKWVRwpYMLJow26e60SaJlanAq/wUXZ9Mc9Z//lDG3oHRz6AkOAzdA/F1Bzo/jj0WYdQok00v1u3V32uT1JSDd8yfsjxjb8k0KR355LcIOSF3XPgbzStewd5nuJVpbtdtRLSVTDKMHHs25g9XIbsckaAQ1JL6iU9dHRqyQGOuiQqn8R+mSqZ91WEFza5TGRDKEvVSWu32IyK+XarHbMWEirIClDUMo36QJRIdZ7krZBVNvyJsby81pACwXm6fyrW0RHUWiEv/6Is4F2ZuaPnJyIXY3rG+IQal3jIePrCMeHpnLjcKBTYeKLcDcTUJvuRJ2lcP09SXAY8ULFDKpuJ3cYkvdT9wto0WhZKcbYcc3R/Pyt1p1hRKAWRABLUiL1yUGsYtkNCod7oxDeC3gh2eZXERyeBDB4aNzIjWitkm+nWSYDr55J8uLWA4TR1GpHt2dNtc04lNU7HIGlz8XvIMwKqX6xdvjVG+6Fuo0n86jyU0t6x1vmHiuxny6lpapYsT/VtxOFMDMBCRju2hKWBalHLkG2ECmLIcsxXqvl5ap6t1IZ17sMjAM+xdWiA2+0mF/CS+2Rd9HoGCn+9EjVEtuuwuhpjRffnXkxOFUiPoGdscRi5phz8TEiz114hC4yRd7pM448v3g7apw2eS3mzOo93ppvAJ2fPCoqN7D7OwyX4j3Nxn8DlvsZg5XP/mtkvUo4eN9qkfhD55QhgoxiVIPn1OSB148L6hV9K4nol6v9y+Dv3+Dy36jLqOBggQwJBnTkMTBZR8+Bpf9eIA8O71+o3FyJy6OI5QKZP+vAZtyd9LIHSBte3cinGitNHNqrtvJHbMNn3x3wscw6dqTpu0c7Z0jlSyJSyd6RoMlzbnOsvtU59WtrAf8bC1548FL8vYAv+bznda74uMT0Rsv4Jlqy6z3magp89s2IPqab9s5RFhbpal0P+erWf7UVqbV/18Bzdb2fc2XBwoPs1J72xsv8r5++sd/iw3PuE/z/cdB/Iz7hfiOZaKHLyD8Qzgd9NUvdf8kqp/0V9gAqh/RVtgAKhpV2AAqGlXYACoaVSAfx397cjcEmntnpQAAAABJRU5ErkJggg==" alt="image"></p>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ul style="margin-bottom:0cm;list-style-type: square;margin-left: 26px;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>Otherwise, activate conda:</li>
    </ul>
</div>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 415.6pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;height: 19.1pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;'><span style='font-size:13px;font-family:"Courier New";'>conda activate</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;'><strong><em><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>&nbsp;</span></em></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;'><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>1.3 ONT Minknow</span></strong><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>&nbsp;and Guppy Installation</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>Depending on whether the user is basecalling on a CPU or a GPU, you will need to choose an appropriate Guppy package. These packages are available as separate installers on the&nbsp;</span><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>Oxford</span><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>&nbsp;Nanopore&nbsp;</span><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>Technologies</span><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>&nbsp;community website (</span><a href="https://community.nanoporetech.com"><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>https://community.nanoporetech.com</span></a><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>).</span></p>
<ul class="decimal_type" style="list-style-type: square;">
    <li><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>CPU-only basecalling is available in the&nbsp;</span><span style='font-family:"Courier New";'>ont-guppy-cpu</span><span style='font-family:"Times New Roman",serif;'>&nbsp;</span><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>package.</span></li>
    <li><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>GPU basecalling is available in the&nbsp;</span><span style='font-family:"Courier New";'>ont-guppy</span><span style='font-family:"Times New Roman",serif;'>&nbsp;</span><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>package and will require a GPU driver for basecalling to work.&nbsp;</span><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;color:#222222;'>To install the GPU version of Guppy software for quicker basecalling, access the Software Downloads section on the Oxford Nanopore Technologies Community site (</span><a href="http://www.community.nanoporetech.com"><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>www.community.nanoporetech.com</span></a><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;color:#222222;'>).</span></li>
    <li><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>Login to&nbsp;</span><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;color:#222222;'>Oxford</span><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>&nbsp;Nanopore community website (https://community.nanoporetech.com) using nanopore login credential, login id and password (</span><a href="https://community.nanoporetech.com/"><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>https://community.nanoporetech.com/</span></a><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>).</span></li>
    <li><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>Direct to &ldquo;Software downloads&rdquo; page from right selection options (</span><a href="https://community.nanoporetech.com/downloads"><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>https://community.nanoporetech.com/downloads</span></a><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>).</span></li>
    <li><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>Download Guppy (latest v6.5.7) for Linux 64-bit GPU download option.</span></li>
    <li><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>Install Guppy to GPU enabled server by extracting the downloaded file named&nbsp;</span><span style='font-family:"Courier New";'>ont-guppy_6.5.7_linux64.tar</span><span style='line-height:107%;font-family:"Times New Roman",serif;font-size:16px;'>, which is typically a tarball archive file, with the below command,</span></li>
</ul>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 451.05pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;'><span style='font-size:13px;font-family:"Courier New";'>tar -xf ont-guppy_6.5.7_linux64.tar</span><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>&nbsp;</span></p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman",serif;'># Inside the&nbsp;</span><span style='font-family:"Courier New";'>.bash_profile</span><span style='font-family:  "Times New Roman",serif;'>&nbsp;</span><span style='font-size:16px;font-family:  "Times New Roman",serif;'>or&nbsp;</span><span style='font-family:"Courier New";'>bashrc</span><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;file, add the following line to append the bin folder containing Guppy executables as shown below to system&apos;s PATH variable.</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style="font-family:Consolas;"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAMIAAAAcCAIAAAC8t/c5AAAACXBIWXMAAAsTAAALEwEAmpwYAAAAtGVYSWZJSSoACAAAAAYAEgEDAAEAAAABAAAAGgEFAAEAAABWAAAAGwEFAAEAAABeAAAAKAEDAAEAAAACAAAAEwIDAAEAAAABAAAAaYcEAAEAAABmAAAAAAAAAEgAAAABAAAASAAAAAEAAAAGAACQBwAEAAAAMDIxMAGRBwAEAAAAAQIDAACgBwAEAAAAMDEwMAGgAwABAAAA//8AAAKgBAABAAAAwgAAAAOgBAABAAAAHAAAAAAAAADNp1TQAAAF1ElEQVR4nO1avU7rShBep05eIKliRYgy6Ull0RCJGhehQykQzXFPFFrk0yCKtKQwHRJSaJAr0sclQkemCg9wXmCvZnZt79rrjQ25EA7+KsLsz+zMNz+7iUEpJRUqfAy1D86vUIEQYtTr9a/WocK3R5WNKmwAFY0qbAAVjSpsmkb27G0x7sUfe+PF29vMNgzy6YCtF+PeV2z9ERiGPXuLMLPJD6SRPXtzrfDxPiBbgOD+MTRH85sjsn0sESNNktqz1colTqvZbLZag2nb/TlM4jQy7JlrEd/pT5Zb8YxEl5O+4xPLndnfIyEZhn3jWr7TGnpgQEqXF33Ht04ho4opikFOtAam/RTEg6cGiCJcOqkYfCSur1w23j5We6ZbOVclBY2YCYjvMBNsCag3RCLdfElVLYsjJJFoQEq9B9/cP+yyj+F00ERAoiKj+eop9iWDj2ksRrwUJLn5iETTm83mw0F6Lh9p2Dfzkek7zb3JklK6nOzFO4aoQYttAeI4fbZjxRzfcldyrvUdPqU5mIYZqUyj7vkpkOj4NsdE3YTVcpbWREmG6UkERFJxclrKcPt7GhLr9Jx7oghwVVhNWF1aXBN/GilbbbVyLULM0VzUm+CsAyuc/k4b8PlPaHZ2U//ERDWYhubo6nxt82cYPXBOOD27SJoNb8h5oKCyzo+qlX1nb7LkinlDVOuXMm6Xkz3HJ3nSmmH0DvdN4j946m9FLHc171xjvUdCxnwE7l+RM4mtSZSAdAUBGoXgdUewGkrno1cuZeGZzZl0Obn2ibl/WLLXNkdzUDoKxMRhRm/8JMQfi7B4X42UBXar5fhCXmFxTcCBB6yrTLXYwcsrae9kg4DS5f1jCAcrGCBZMmbQGy9cK5wOjnP8mEGXuf023ZIS60DXkarOA9kIlwv/POfNS+q97FRKWVRwpYMLJow26e60SaJlanAq/wUXZ9Mc9Z//lDG3oHRz6AkOAzdA/F1Bzo/jj0WYdQok00v1u3V32uT1JSDd8yfsjxjb8k0KR355LcIOSF3XPgbzStewd5nuJVpbtdtRLSVTDKMHHs25g9XIbsckaAQ1JL6iU9dHRqyQGOuiQqn8R+mSqZ91WEFza5TGRDKEvVSWu32IyK+XarHbMWEirIClDUMo36QJRIdZ7krZBVNvyJsby81pACwXm6fyrW0RHUWiEv/6Is4F2ZuaPnJyIXY3rG+IQal3jIePrCMeHpnLjcKBTYeKLcDcTUJvuRJ2lcP09SXAY8ULFDKpuJ3cYkvdT9wto0WhZKcbYcc3R/Pyt1p1hRKAWRABLUiL1yUGsYtkNCod7oxDeC3gh2eZXERyeBDB4aNzIjWitkm+nWSYDr55J8uLWA4TR1GpHt2dNtc04lNU7HIGlz8XvIMwKqX6xdvjVG+6Fuo0n86jyU0t6x1vmHiuxny6lpapYsT/VtxOFMDMBCRju2hKWBalHLkG2ECmLIcsxXqvl5ap6t1IZ17sMjAM+xdWiA2+0mF/CS+2Rd9HoGCn+9EjVEtuuwuhpjRffnXkxOFUiPoGdscRi5phz8TEiz114hC4yRd7pM448v3g7apw2eS3mzOo93ppvAJ2fPCoqN7D7OwyX4j3Nxn8DlvsZg5XP/mtkvUo4eN9qkfhD55QhgoxiVIPn1OSB148L6hV9K4nol6v9y+Dv3+Dy36jLqOBggQwJBnTkMTBZR8+Bpf9eIA8O71+o3FyJy6OI5QKZP+vAZtyd9LIHSBte3cinGitNHNqrtvJHbMNn3x3wscw6dqTpu0c7Z0jlSyJSyd6RoMlzbnOsvtU59WtrAf8bC1548FL8vYAv+bznda74uMT0Rsv4Jlqy6z3magp89s2IPqab9s5RFhbpal0P+erWf7UVqbV/18Bzdb2fc2XBwoPs1J72xsv8r5++sd/iw3PuE/z/cdB/Iz7hfiOZaKHLyD8Qzgd9NUvdf8kqp/0V9gAqh/RVtgAKhpV2AAqGlXYACoaVSAfx397cjcEmntnpQAAAABJRU5ErkJggg==" alt="image" width="194" height="28"></span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>nano ~/.bash_profile</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>export PATH=&quot;/path/to/ont-guppy/bin:$PATH&quot;</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman",serif;'># After saving the changes, apply them to your current terminal session by the following command to reload the configuration.</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>source ~/.bash_profile</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;'><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>&nbsp;</span></p>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ol start="2" style="margin-bottom:0cm;list-style-type: decimal;margin-left:0px;padding-left: 0;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:15.0pt;font-family:"Calibri",sans-serif;'><strong><em><span style='font-family:"Times New Roman",serif;font-size:20px;color:black;'>Basecalling and Demultiplexing</span></em></strong></li>
    </ol>
</div>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;border:none;'><em><span style='font-size:16px;font-family:"Times New Roman";color:black;'>&nbsp;</span></em></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>2.1 Basecalling using <em>guppy_basecaller</em>&nbsp;</span></strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>(High-Accuracy Model&nbsp;</span><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>using GPU version)</span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>To convert the raw signal data (FAST5) from the nanopore sequencer into readable nucleotide sequences (FASTQ) we will perform basecalling using Guppy Basecaller using the GPU version with the following commands.&nbsp;</span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>Note</span></strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>: In case of Oxford Nanopore GridION sequencing device, if basecalling has been performed with itself from device, then this step can be skipped and directly <em>Demultiplexing</em> can be done.&nbsp;</span></p>
<pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'># <u>Without flowcell and kit name</u>:</span></pre>
<pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'> </span></pre>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0.5cm 5.4pt;vertical-align: top;">
                <pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";line-height:115%;'><span style="color:black;">guppy_basecaller -c dna_r10.4.1_e8.2_400bps_hac.cfg -i /path/to/Runfolder/ -s ./basecalling/ -x auto </span></pre>
                <pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";line-height:115%;'><span style="color:black;"># The auto can be replaced with CUDA cores for GPU-basecalling using,</span></pre>
                <pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";line-height:115%;'><span style="color:black;">guppy_basecaller -c dna_r10.4.1_e8.2_400bps_hac.cfg -i /path/to/Runfolder/ -s ./basecalling/ -x &quot;cuda:1&quot; -r</span></pre>
            </td>
        </tr>
    </tbody>
</table>
<pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'> </span></pre>
<pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'># <u>With flowcell and kit name</u>:</span></pre>
<pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'> </span></pre>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="border: 1pt solid windowtext; padding: 0.5em; vertical-align: top; width: 100%;">
                <pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";line-height:115%;'><span style="color:black;">guppy_basecaller -c dna_r10.4.1_e8.2_400bps_hac.cfg -i /path/to/Runfolder/ -s ./basecalling/ --flowcell FLO-MIN114  </span></pre>
                <pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";line-height:115%;'><span style="color:black;"> --kit SQK-NBD114-24 -x auto -r </span></pre>
				<pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";line-height:115%;'><span style="color:black;"># The auto can be replaced with CUDA cores for GPU-basecalling using,</span></pre>
                <pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";line-height:115%;'><span style="color:black;">guppy_basecaller -c dna_r10.4.1_e8.2_400bps_hac.cfg -i /path/to/Runfolder/ -s ./basecalling/ --flowcell FLO-MIN114 </span></pre>
				<pre style='margin:0cm;margin-bottom:.0001pt;font-size:13px;font-family:"Courier New";line-height:115%;'><span style="color:black;"> --kit SQK-NBD114-24 -x &quot;cuda:1&quot; -r</span></pre>
            </td>
        </tr>
    </tbody>
</table>



<ul class="decimal_type" style="list-style-type: disc;margin-left: 25.450000000000003px;">
    <li><span style="font-family:Consolas;font-size:15px;color:black;">-c:</span><span style='font-family:"Times New Roman",serif;font-size:15px;color:black;'>&nbsp;</span><span style='font-family:"Times New Roman",serif;font-size:16px;color:black;'>Configuration file</span></li>
    <li><span style="font-family:Consolas;font-size:15px;color:black;">-x:</span><span style='font-family:"Times New Roman",serif;font-size:15px;color:black;'>&nbsp;</span><span style='font-family:"Times New Roman",serif;font-size:16px;color:black;'>Specify GPU device.</span></li>
    <li><span style="font-family:Consolas;font-size:15px;color:black;">-r:</span><span style='font-family:"Times New Roman",serif;font-size:15px;color:black;'>&nbsp;</span><span style='font-family:"Times New Roman",serif;font-size:16px;color:black;'>Search for input file recursively.</span></li>
    <li><code style='font-family:"Courier New";'><span style='font-family:"Times New Roman",serif;font-size:16px;color:#333333;'>Substitute</span></code><code style='font-family:"Courier New";'><span style='font-family:"Times New Roman",serif;font-size:15px;color:#333333;'>&nbsp;</span></code><span style="font-family:Consolas;color:black;">/path/to/Runfolder/&nbsp;</span><code style='font-family:"Courier New";'><span style='font-family:"Times New Roman",serif;font-size:16px;color:#333333;'>with folder where FAST5 files are present.</span></code></li>
</ul>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:12.0pt;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>2.2 Demultiplexing using&nbsp;</span></strong><strong><em><span style='font-family:"Times New Roman";'>guppy_barcode&nbsp;</span></em></strong><span style='font-size:16px;line-height:107%;font-family:"Times New Roman";color:#222222;'>(using GPU version)</span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>As barcodes are used during library preparation, demultiplexing the reads into separate files for each barcode is required. We will use Guppy Barcoder here. This step will separate the reads belonging to different samples or barcodes.&nbsp;</span></p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;'><span style='font-family:"Courier New";'>guppy_barcoder -i ./basecalling/ -s ./demultiplexing/ &ndash;barcode_kits &quot;EXP-NBD114-24&quot; -x auto</span></p>
            </td>
        </tr>
    </tbody>
</table>
<ul style="list-style-type: disc;margin-left: 11.3px;">
    <li><span style='font-family:"Courier New";font-size:13px;'>-i ./basecalling/</span><span style='font-family:"Times New Roman",serif;font-size:16px;'>: The input directory containing the basecalled reads (FASTQ files).</span></li>
    <li><span style='font-family:"Courier New";font-size:13px;'>-s ./demultiplexing/</span><span style='font-family:"Times New Roman",serif;font-size:16px;'>: The output directory where demultiplexed reads will be saved.</span></li>
    <li><span style='font-family:"Courier New";font-size:13px;'>--barcode_kits &quot;EXP-NBD114-24&quot;</span><span style='font-family:"Times New Roman",serif;font-size:16px;'>: Specifies the barcode kit (in this case, EXP-NBD114).</span></li>
    <li><span style='font-family:"Courier New";'>-x auto:</span><span style='font-family:"Times New Roman",serif;font-size:16px;'>&nbsp;Uses GPU acceleration if available&nbsp;</span><span style='font-family:"Courier New";font-size:13px;'>(-x auto</span><span style='font-family:"Times New Roman",serif;font-size:19px;'>&nbsp;</span><span style='font-family:"Times New Roman",serif;font-size:16px;'>will automatically select the best available hardware)</span></li>
</ul>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ol start="3" style="margin-bottom:0cm;list-style-type: decimal;margin-left: 4.199999999999999px;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><strong><em><span style='font-family:"Times New Roman",serif;font-size:16px;color:black;'>Quality Control and Read Filtering</span></em></strong></li>
    </ol>
</div>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>The raw reads need to be filtered fir low-quality reads, adapters, and barcodes.</span></p>
<h3 style='margin-top:14.0pt;margin-right:0cm;margin-bottom:4.0pt;margin-left:0cm;font-size:19px;font-family:"Calibri",sans-serif;font-weight:bold;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>3.1 Assessing Sequencing Statistics using NanoStat&nbsp;</span></h3>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;'>Finally, generate a summary of the sequencing run using <strong>NanoStat</strong>.</p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <pre style='margin:0cm;font-size:13px;font-family:"Courier New";'>
<code style='font-family:"Courier New";'>NanoStat --fastq filtered_reads.fastq.gz --outdir stats_output/</code>

</pre>
            </td>
        </tr>
    </tbody>
</table>
<ul style="list-style-type: disc;margin-left: 25.450000000000003px;">
    <li><code style='font-family:"Courier New";'><span style='font-family:"Times New Roman",serif;font-size:16px;'>--fastq</span></code><span style='font-family:"Times New Roman",serif;font-size:16px;'>: Input filtered FASTQ file.</span></li>
    <li><code style='font-family:"Courier New";'><span style='font-family:"Times New Roman",serif;font-size:16px;'>--outdir</span></code><span style='font-family:"Times New Roman",serif;font-size:16px;'>: Directory for saving the sequencing statistics</span></li>
</ul>
<h3 style='margin-top:14.0pt;margin-right:0cm;margin-bottom:4.0pt;margin-left:0cm;font-size:19px;font-family:"Calibri",sans-serif;font-weight:bold;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>3.2 Adapter Trimming using Porechop</span></h3>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;'>After basecalling, trim the adapter sequences from the FASTQ files using <strong>Porechop</strong>.</p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <pre style='margin:0cm;font-size:13px;font-family:"Courier New";'><code style='font-family:"Courier New";'>porechop -i basecalled_output/reads.fastq.gz -o trimmed_reads.fastq.gz --threads 4</code></pre>
            </td>
        </tr>
    </tbody>
</table>
<ul type="disc" style="margin-bottom:0cm;">
    <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;line-height:normal;'><code style='font-family:"Courier New";'><span style="font-family:     Calibri;">-i</span></code><span style='font-size:16px;font-family:"Times New Roman",serif;'>: Input basecalled reads in FASTQ format.</span></li>
    <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;line-height:normal;'><code style='font-family:"Courier New";'><span style="font-family:     Calibri;">-o</span></code><span style='font-size:16px;font-family:"Times New Roman",serif;'>: Output file for trimmed reads.</span></li>
    <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;line-height:normal;'><code style='font-family:"Courier New";'><span style="font-family:     Calibri;">--threads</span></code><span style='font-size:16px;font-family:     "Times New Roman",serif;'>: Number of CPU threads for faster processing.</span></li>
</ul>
<h3 style='margin-top:14.0pt;margin-right:0cm;margin-bottom:4.0pt;margin-left:0cm;font-size:19px;font-family:"Calibri",sans-serif;font-weight:bold;'><span style='font-size:16px;line-height:107%;font-family:"Times New Roman",serif;'>3.3 Low-Quality Reads Filtering using NanoFilt</span></h3>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;'>Use <strong>NanoFilt</strong> to filter out reads with a Phred score below 9 and sequences shorter than 200 bp.</p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <pre style='margin:0cm;font-size:13px;font-family:"Courier New";'><code style='font-family:"Courier New";'>zcat trimmed_reads.fastq.gz | NanoFilt -q 9 -l 200 | gzip &gt; filtered_reads.fastq.gz</code></pre>
            </td>
        </tr>
    </tbody>
</table>
<ul type="disc" style="margin-bottom:0cm;">
    <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;line-height:normal;'><code style='font-family:"Courier New";'><span style="font-size:16px;font-family:Calibri;">zcat</span></code><span style='font-size:16px;font-family:"Times New Roman",serif;'>: Decompresses the <code style='font-family:"Courier New";'><span style="font-family:Calibri;">.fastq.gz</span></code> file.</span></li>
    <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;line-height:normal;'><code style='font-family:"Courier New";'><span style="font-size:16px;font-family:Calibri;">-q 9</span></code><span style='font-size:16px;font-family:"Times New Roman",serif;'>: Filters out reads with a Phred quality score below 9.</span></li>
    <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;line-height:normal;'><code style='font-family:"Courier New";'><span style="font-size:16px;font-family:Calibri;">-l 200</span></code><span style='font-size:16px;font-family:"Times New Roman",serif;'>: Removes reads shorter than 200 bp.</span></li>
    <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;color:#222222;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>The output is recompressed and saved as <code style='font-family:"Courier New";'><span style="font-family:Calibri;">filtered_reads.fastq.gz</span></code>.</span></li>
</ul>
<div style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'>
    <ol start="4" style="margin-bottom:0cm;list-style-type: decimal;margin-left: 4.199999999999999px;">
        <li style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;font-size:11.0pt;font-family:"Calibri",sans-serif;'><strong><em><span style='font-family:"Times New Roman",serif;font-size:16px;color:black;'>AMR Detection</span></em></strong><strong><em><span style='font-family:"Times New Roman",serif;font-size:16px;color:#222222;'>&nbsp;using&nbsp;</span></em></strong><strong><em><span style='font-family:"Times New Roman",serif;font-size:16px;color:black;'>ResFinder&nbsp;</span></em></strong></li>
    </ol>
</div>
<p style='margin-right:0cm;margin-left:14.2pt;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>Provides a list of identified AMR genes, the level of coverage and identity, and statistics on the matches. It relies on the ResFinder database, which contains curated sequences of resistance genes for a wide range of antibiotic classes.</span></p>
<p style='margin-right:0cm;margin-left:-3.8pt;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>4.1 ResFinder Installation and Dependencies</span></strong></p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;background:white;'><span style='font-size:13px;font-family:"Courier New";color:#767171;'># Clone the latest version and enter the resfinder directory</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;background:white;'><span style='font-size:13px;font-family:"Courier New";color:black;'>git clone https://github.com/cadms/resfinder.git</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>cd resfinder &amp;&amp; make</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>Depending on how you plan to run ResFinder, either BLAST or KMA options can be used. BLAST is used to analyse assemblies (i.e. FASTA files). KMA is used to analyse read data (i.e. FASTQ files).</span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>4.1.1 Python modules: Tabulate, BioPython, CGECore and Python-Git</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>To install the needed python modules you can use pip.</span></p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;background:white;'><span style='font-size:13px;font-family:"Courier New";color:black;'>pip3 install tabulate biopython cgecore gitpython python-dateutil</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>4.2 KMA</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-family:"Times New Roman",serif;color:black;'>KMA is a mapping method designed to map raw reads directly against redundant databases. The instructions here will install KMA in the default location ResFinder uses. KMA can be installed in another location but the path to KMA will then need to be specified every time you run ResFinder unless you add the kma program to your PATH.</span></p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;background:white;'><span style='font-size:13px;font-family:"Courier New";color:#767171;'># Go to the directoy in which you installed the ResFinder tool</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;background:white;'><span style='font-size:13px;font-family:"Courier New";color:black;'>cd /path/to/some/dir/resfinder</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;background:white;'><span style='font-size:13px;font-family:"Courier New";color:black;'>cd cge</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;background:white;'><span style='font-size:13px;font-family:"Courier New";color:black;'>git clone https://bitbucket.org/genomicepidemiology/kma.git</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;background:white;'><span style='font-size:13px;font-family:"Courier New";color:black;'>cd kma &amp;&amp; make</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p><br></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>4.3</span></strong><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>&nbsp;<strong>Databases</strong></span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>This section describes how to install the databases at the ResFinder default locations. The database locations can be changed, but must then be specified to ResFinder at run time.</span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>4.3.1 ResFinder database</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>Provides matches of resistance genes with details like identity, coverage, and alignment scores.</span></p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";color:#767171;'># Go to the directory in which you installed the ResFinder tool</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>cd /path/to/dir/resfinder</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>git clone https://git@bitbucket.org/genomicepidemiology/resfinder_db.git db_resfinder</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";color:#767171;'># Indexing databases</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>cd db_resfinder</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>python3 INSTALL.py</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:8.0pt;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span><strong><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>4.3.2 PointFinder database</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>Detects specific point mutations within genes that are known to cause antimicrobial resistance.</span></p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";color:#767171;'># Go to the directory in which you installed the ResFinder tool</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>cd /path/to/dir/resfinder</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>git clone https://git@bitbucket.org/genomicepidemiology/pointfinder_db.git db_pointfinder</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";color:#767171;'># Indexing databases</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>cd db_pointfinder</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:13px;font-family:"Courier New";'>python3 INSTALL.py</span></p>
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:  normal;'><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>4.4 AMR Detection</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;color:black;'>Command you are using with KMA (k-mer alignment tool) for antimicrobial resistance (AMR) detection in Nanopore reads.</span></p>
<table style="border-collapse: collapse; border: none; width: 100%; margin-right: calc(7%);">
    <tbody>
        <tr>
            <td style="width: 450.8pt;border: 1pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;background:  white;'><span style='font-family:"Courier New";color:black;'>kma -i nanoporeReads.fq.gz -o amr_out -t_db ./resfinder/db_resfinder/all -mem_mode -mp 20 -mrs 0.0 -bcNano -bc 0.7</span></p>
            </td>
        </tr>
    </tbody>
</table>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></p>
<p style='margin-right:0cm;margin-left:1.0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman";'>&middot;</span><span style='font-size:16px;font-family:"Times New Roman";'>&nbsp;&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";'>-i nanoporeReads.fq.gz</span><span style='font-size:16px;font-family:"Times New Roman";'>: Input file containing the raw Nanopore reads in FASTQ format.</span></p>
<p style='margin-right:0cm;margin-left:1.0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman";'>&middot;</span><span style='font-size:16px;font-family:"Times New Roman";'>&nbsp;&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";'>-o amr_out</span><span style='font-size:16px;font-family:"Times New Roman";'>: Output file prefix (results will be saved as&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";'>amr_out.*</span><span style='font-size:16px;font-family:"Times New Roman";'>).</span></p>
<p style='margin-right:0cm;margin-left:1.0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman";'>&middot;</span><span style='font-size:16px;font-family:"Times New Roman";'>&nbsp;&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";'>-t_db ./resfinder/db_resfinder/all</span><span style='font-size:16px;font-family:"Times New Roman";'>: Specifies the ResFinder AMR database.</span></p>
<p style='margin-right:0cm;margin-left:1.0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman";'>&middot;</span><span style='font-size:16px;font-family:"Times New Roman";'>&nbsp;&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";'>-mem_mode</span><span style='font-size:16px;font-family:"Times New Roman";'>: Enables memory mode, suitable for large datasets.</span></p>
<p style='margin-right:0cm;margin-left:1.0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman";'>&middot;</span><span style='font-size:16px;font-family:"Times New Roman";'>&nbsp;&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";'>-mp 20</span><span style='font-size:16px;font-family:"Times New Roman";'>: Sets the minimum posterior value to 20 for the mapping.</span></p>
<p style='margin-right:0cm;margin-left:1.0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman";'>&middot;</span><span style='font-size:16px;font-family:"Times New Roman";'>&nbsp;&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";'>-mrs 0.0</span><span style='font-size:16px;font-family:"Times New Roman";'>: Sets minimum relative score to 0.0, allowing all reads to be considered.</span></p>
<p style='margin-right:0cm;margin-left:1.0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;line-height:normal;'><span style='font-size:16px;font-family:"Times New Roman";'>&middot;</span><span style='font-size:16px;font-family:"Times New Roman";'>&nbsp;&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";'>-bcNano</span><span style='font-size:16px;font-family:"Times New Roman";'>: Specifically optimized for Nanopore reads, taking into account their error profile.</span></p>
<p style='margin-right:0cm;margin-left:1.0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><span style='font-size:16px;font-family:"Times New Roman";color:black;'>&middot;</span><span style='font-size:16px;font-family:"Times New Roman";color:black;'>&nbsp;&nbsp;</span><span style='font-size:13px;font-family:"Times New Roman";color:black;'>-bc 0.7</span><span style='font-size:16px;font-family:"Times New Roman";color:black;'>: Sets the minimum coverage to 70% for reporting a hit.</span></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></strong></p>
<p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin-top:0cm;margin-bottom:0cm;font-size:11.0pt;text-align:justify;line-height:normal;background:white;'><strong><span style='font-size:16px;font-family:"Times New Roman",serif;'>&nbsp;</span></strong></p>

 <footer>
        <p>Contributed by Ranjeet Maurya and Priyanka Mehta <br> c/o Dr Rajesh Pandey</p>
    </footer>
</body>

</html>
